StigData/Processed/InternetExplorer-11-1.17.org.default.xml

```diff
@@ -0,0 +1,8 @@
+<!--
+    The organizational settings file is used to define the local organizations
+    preferred setting within an allowed range of the STIG.
+
+    Each setting in this file is linked by STIG ID and the valid range is in an
+    associated comment.
+-->
+<OrganizationalSettings fullversion="1.17" />
```

Comment:
The test will probably fail without a new line at the end of this file.

Reply:
Caught that and corrected.